x-pack/plugin/inference/src/main/java/org/elasticsearch/xpack/inference/services/elastic/ElasticInferenceService.java

```diff
@@ -385,6 +235,7 @@ protected void doInfer(
                 responseString = responseString + " " + useChatCompletionUrlMessage(model);
             }
             listener.onFailure(new ElasticsearchStatusException(responseString, RestStatus.BAD_REQUEST));
+            return;
```

Comment:
Noticed we had a fall through bug that luckily is getting picked up by the if-block below but figured I'd fix it anyway.